src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -169,6 +169,14 @@ export default class TradeStore extends BaseStore {
         WS.forgetAll('proposal');
     };
 
+    @action.bound
+    selectedSymbolExchangeStatus = (activeSymbols) => {
```

Comment:
I suggest renaming it, since it doesn't really tell what it does. 
```
    isSymbolEchangeOpen = (activeSymbols) => `true|false`
```

Although, you can go with a `@computed` property, but that requires persisting `active_symbols`.